admin/admin.php

```diff
@@ -31,6 +31,8 @@ public function __construct( $plugin ) {
 	public function register_hooks() {
 		global $wp_version;
 
+		add_action( 'admin_enqueue_scripts', array( &$this, 'thickbox' ) );
```

Comment:
May be able to remove the `&` character in this scenario. 

Might even be able to just do `add_action( 'admin_enqueue_scripts', 'add_thickbox' );` if thats the only function you want to run.